USFMConverter/UI/Pages/FileView.axaml.cs

```diff
@@ -277,5 +307,11 @@ public void UpdateSelectBtn()
             selectAllBtn.Content = anySelected ? "Unselect All" : "Select All";
             selected = anySelected;
         }
+
+        private void OnSortFiles(object sender, RoutedEventArgs e)
+        {
+            var files = filesContainer.Items.Cast<string>().ToList();
+            filesContainer.Items = files.OrderBy(f => new FileInfo(f).Name).ToList();
```

Comment:
I think you should use FileInfo.FullPath here, not FileInfo.Name.  In BTT-Writer repos, each verse is named with just its verse number, e.g. 10.txt.  Sorting by FileInfo.Name would have all the 10s together, then the 11s, and so forth!

Reply:
@PurpleGuitar I did try and see that FileInfo.Name is the full file name, not stripping off the extension (see screenshot below)

![image](https://user-images.githubusercontent.com/34975907/126552511-dc2e430d-b437-4d7a-a01f-845bd4d4bfca.png)

On the other hand, FileInfo.FullName gives us the absolute path of the file/direction instead of the name alone. 

![image](https://user-images.githubusercontent.com/34975907/126552789-b097e4b9-cbc4-44da-9ae8-4e4882070320.png)

Reply:
OK, thanks for checking.  I think `FullName` will work fine then instead of `Name`, to make sure that we sort by directory, not just the short filename.